code/clue.js

```diff
@@ -1,59 +1,237 @@
+  image: 'assets/ra.jpg'
+}
 
-const rooms = []
+const ChandlerDiningRoom = {
+  name: "Chandler's Dining Room",
+  image: 'assets/cdr.jpg'
+}
+
+const RachelRoom = {
+  name: "Rachel's Room",
+  image: 'assets/rbr.jpg'
+}
+
+const MonicaRoom = {
+  name: "Monica's Room",
+  image: 'assets/mbr.png'
+}
+
+const MonicaLivingRoom = {
+  name: "Monica's Living room",
+  image: 'assets/mlr.jpg'
+}
+
+const CentralPerk = {
+  name: "Central Perk",
+  image: 'assets/cp.jpg'
+}
+
+const Balcony = {
+  name: "the Balcony",
+  image: 'assets/balcony.jpg'
+}
+
+const Hall = {
+  name: "the Hall",
+  image: 'assets/stairwell.jpg'
+}
+
+const ChandlerLivingRoom = {
+  name: "Chandler's Living Room",
+  image: 'assets/clr.jpg'
+}
+
+const friend = [
+  Ross,
+  Chandler,
+  Monica,
+  Rachel,
+  Phoebe,
+  Marcel
+];
+
+const foods = [
+  Hamburger,
+  RachelTrifle,
+  Turkey,
+  RedBagel,
+  Pizza,
+  Muffin,
+  KiwiPie,
+  Sandwich,
+];
+
+const rooms = [
+  MonicaDiningRoom,
+  RossApartment,
+  ChandlerDiningRoom, 
+  RachelRoom, 
+  MonicaRoom, 
+  MonicaLivingRoom, 
+  CentralPerk, 
+  Balcony, 
+  Hall, 
+  ChandlerLivingRoom
+]
 
-// THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FROM THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
-// YOU DON'T NEED TO CHANGE THIS, JUST TRY TO UNDERSTAND IT. AND HOW TO USE IT.
 const randomSelector = array => {
   return array[Math.floor(Math.random() * array.length)]
 }
 
-// CREATE AN OBJECT THAT KEEPS THE MYSTERY.
-// With a killer, a weapon and a room.
-// The values will be set later.
+const mystery = {
+  friend: undefined,
+  foods: undefined,
+  rooms: undefined
+}
+
+const pickFriends = () => {
+  mystery.friends = randomSelector(friend)
+  document.getElementById('friendsName').innerHTML = `${mystery.friends.firstName} ${mystery.friends.lastName}`;
+  document.getElementById('friendsAge').innerHTML = `${mystery.friends.age}`;
+  document.getElementById('friendsOccupation').innerHTML = mystery.friends.occupation;
+  document.getElementById('friendsDescription').innerHTML = mystery.friends.description;
+  document.getElementById('friendsImage').src = mystery.friends.image
+}
 
-// This function will be invoked when you click on the killer card.
-const pickKiller = () => {
-  // This will randomly select a killer from the suspects. And add that to the mystery object.
-  mystery.killer = randomSelector(suspects)
+document.getElementById('friendsCard').onclick = pickFriends
 
-  // This will change the background color of the card to the one connected to the chosen killer and show the full name of the killer. Feel free to add more things to show about the killer.
-  document.getElementById('killerCard').style.background = mystery.killer.color
-  document.getElementById(
-    'killerName'
-  ).innerHTML = `${mystery.killer.firstName} ${mystery.killer.lastName}`
+const pickFoods = () => {
+  mystery.foods = randomSelector(foods)
+  document.getElementById('foodsName').innerHTML = mystery.foods.name;
```

Comment:
For having a more clean code u should stick with writing the code with the ${ } as in 224 in every code that writing like that. For a better structure ❤️

Reply:
Thank you Destiny, I am going to fix it!(:

Reply:
Not really - we should use template strings syntax (`${}`) only when we want to combine variable with fixed string, like in 
`document.getElementById('foodsName').innerHTML = 'Mystery food was ${mystery.foods.name;}'`

On the other hand, if you only need to pass variable without any fixed text like `"Mystery food was"`, then you can (and you even should) just write :
`document.getElementById('foodsName').innerHTML = mystery.foods.name;`